docs/user_guide.rst

```diff
@@ -16,8 +16,15 @@ Reading genetic data
 Installation
 ------------
 
-Sgkit can read standard genetic file formats, including VCF, PLINK, and BGEN. Support for reading
-these formats is not installed by default, and requires additional dependencies, which can be installed
+Sgkit can read standard genetic file formats, including VCF, PLINK, and BGEN.
+
+If sgkit has been installed using conda, support for reading BGEN and PLINK is included, but
+VCF is not, and should be installed using pip as follows::
+
+    $ pip install 'sgkit[vcf]'
```

Comment:
I'm not sure this is the best advice - why not be up front and say "VCF support is not included because there is no Windows support for cyvcf2, the library we use for reading VCF data. If you are using Linux or a Mac, please install cyvcf2 using ``conda install -c bioconda cyvcf2`` to enable VCF support."

Unless we explain the situation clearly we'll end up answering endless questions from users with failing installations. (I.e., someone on a Mac who doesn't have a working C compiler and tries to install with pip, like above)

Reply:
Thanks for the suggestion - I've updated the text.